x/evm/keeper/rpc_server.go

```diff
@@ -5,74 +5,123 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/core"
 	"github.com/evmos/ethermint/x/evm/statedb"
 )
 
 // EthmRpcServer is a RPC server wrapper around the keeper. It is updated on
 // each new sdk.Message with the latest context and Ethereum core.Message.
 type EthmRpcServer struct {
-	ctx    sdk.Context
-	msg    core.Message
-	evmCfg *EVMConfig
-	k      *Keeper
+	Keeper *Keeper
```

Comment:
Prefer to use private members/methods, unless it needs to be exported.

```suggestion
	keeper *Keeper
```